LoyaltyProgram_Intern/src/main/java/com/pgs/soft/domain/Card.java

```diff
@@ -0,0 +1,60 @@
+package com.pgs.soft.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "card")
+public class Card {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Integer id;
+	
+	private String number;
+	
+	@ManyToOne
+	@JoinColumn(name = "user_id")
+	private User user;
+	
+	@Column(name = "active", columnDefinition = "TINYINT", length = 1)
```

Comment:
There is bit type in sql, please remove columnDefinition and length properties and update table in db